processor/processor.go

```diff
@@ -969,7 +973,7 @@ func (proc *HandleT) getFailedEventJobs(response transformer.ResponseT, commonMe
 
 		for _, message := range messages {
 			sampleEvent, err := jsonfast.Marshal(message)
-			if err != nil {
+			if err != nil || proc.transientSources.Apply(commonMetaData.SourceID) {
 				proc.logger.Errorf(`[Processor: getFailedEventJobs] Failed to unmarshal first element in failed events: %v`, err)
```

Comment:
Is the log message still relevant is we get in here due to `proc.transientSources.Apply(commonMetaData.SourceID)` instead of because of `err != nil`?

Reply:
no it is not relevant, thanks for spotting once again!